src/context/AppProvider/AppContext.tsx

```diff
@@ -168,12 +168,12 @@ export const AppProvider = ({ children }: { children: React.ReactNode }) => {
 
         const accountMetadataByAccountId: AccountMetadataById = {}
         const isMultiAccountWallet = wallet.supportsBip44Accounts()
```

Comment:
Multichain hdwallet was wrongly detecting bip44 (multi-account) support. This is *not* correct, and has been fixed in hdwallet-land.